build_deploy.sh

```diff
@@ -99,4 +103,31 @@ make \
   internal_image_registry="quay.io" \
   image_repository="$QUAY_ORG/kas-fleet-manager" \
   docker/login \
-  image/push
\ No newline at end of file
+  image/push
+
+# create new relaase in managed-kafka-versions repo for kas-fleet-manager
+if [[ ! -z "$AUTHOR_EMAIL" ]] && [[ ! -z "$AUTHOR_NAME" ]] && [[ ! -z "$GITLAB_TOKEN" ]] && \
+[[ ! -z "$PASSWORD" ]] && [[ ! -z "$USERNAME" ]]; then
+  LATEST_COMMIT=$(git rev-parse HEAD)
+  echo "$LATEST_COMMIT"
+  SECOND_LAST_COMMIT=$(git rev-parse HEAD~1)
+  echo "$SECOND_LAST_COMMIT"
+  IMAGE_TAG=$(git rev-parse --short=7 HEAD)
```

Comment:
I think the `VERSION` variable could be re-used here

Reply:
I don't think it can. it has 8 characters. Tag has 7 chars. Unless I change the VERSION to be 7 chars. WDYT?

Reply:
We should be using `VERSION` as the image tag and not computing the image tag again. That's what get's passed in when the image is built and pushed:
- https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/98617fa984036f2dfbcf7966872082b3e9e83e89/build_deploy.sh#L90
- https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/98617fa984036f2dfbcf7966872082b3e9e83e89/Makefile#L27
- https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/98617fa984036f2dfbcf7966872082b3e9e83e89/Makefile#L580
- https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/98617fa984036f2dfbcf7966872082b3e9e83e89/Makefile#L585

Reply:
@machi1990 - if you look at any tag on quay, it contains first 7 chars of the latest commit sha.

Reply:
> I understand it, but if you echo "$VERSION" - it will contain 8 characters. image tag contains 7 (maybe quay only accepts 7 chars?). I can try to change the VERSION to be `$(git rev-parse --short=7 HEAD)` and reuse it for both building and for the places where currently `IMAGE_TAG` is used. WDYT?

+1. Using one variable is much safer.

Reply:
I have changed VERSION and using it instead of IMAGE_TAG

Reply:
Thanks!